src/test/java/capstone/mfslbackend/service/TeamServiceTest.java

```diff
@@ -70,7 +73,8 @@ public void testCreateTeamsInLeague_SuccessTeamDoesntExist() throws IOException
         assertEquals(200, responseEntity.getStatusCode().value()); // Status code should be OK
         List<Team> teams = responseEntity.getBody();
         assertFalse(CollectionUtils.isEmpty(teams)); // Teams should not be empty
-        assertEquals(teamId, teams.get(0).getTeamId());
+        Team actualTeam = teams.get(0);
+        assertEquals(Optional.of(teamId), Optional.ofNullable(actualTeam.getTeamId()));
```

Comment:
why do you put them in optional containers? it shouldn't make any difference at all